Add creating snapshots for a single C file (#279)
viktormalik/diffkemp: README.md

    also has additional options to configure the project build, see `diffkemp
    build --help` for the complete list.

>     The command can be also used for a snapshot generation for a single C file,
>     in this case, the `PROJ_DIR` represents the path to the file. 
>     If `SYMBOL_LIST` is not given, all functions from the file are considered.

Reviewer: Small rewording:
```suggestion
    The command can be also used to generate a snapshot from a single C file.
    In this case, the path to the file should be given in place of `PROJ_DIR`. 
    
    If `SYMBOL_LIST` is not given, all functions from the project/file are considered.
```

Author: I omitted the last line because `SYMBOL_LIST` is already mentioned in the lines above
https://github.com/viktormalik/diffkemp/blob/abab4e4bdf8ec485d944a2d8b552a2a736dff257/README.md?plain=1#L85-L86
